app/js/views/services-list-item.jsx

```diff
@@ -149,21 +147,42 @@ export default class ServicesListItem extends React.Component {
       }
     }
 
-    let isConnected = false;
-    if (this.state.available !== undefined) {
-      isConnected = this.state.available;
-    }
-
     return (
-      <li data-icon={icon} data-connected={isConnected}>
-        <a href={`#services/${this.props.id}`}>
+      <li className="service-list__item" data-icon={icon} data-connected={isConnected}>
+        <a className="service-list__item-link" href={`#services/${this.props.id}`}>
           {serviceType}
           <small>{` (${this.props.name})`}</small>
         </a>
-        <div className="colour-picker" style={{ background: this.getBulbColour.call(this) }}></div>
-        <input type="checkbox" checked={this.state.on} disabled={!isConnected}
-          onChange={this.handleLightOnChange.bind(this)}/>
+        <div className="service-list__item-color-picker"
+             style={{ background: this.getBulbColour.call(this) }}>
+        </div>
+        <label>
+          <input className="service-list__on-off-toggle" type="checkbox"
+                 checked={this.state.on} disabled={!isConnected}
+                 onChange={this.handleLightOnChange.bind(this)} />
+        </label>
       </li>
     );
   }
+
+  renderGenericService(type = 'Unknown service', icon = 'unknown') {
+    return (
+      <li className="service-list__item" data-icon={icon} data-connected="true">
+        <a className="service-list__item-link" href={`#services/${this.props.id}`}>
+          {type} <small>{` (${this.props.name})`}</small>
+        </a>
+      </li>
+    );
+  }
+
+  render() {
+    switch (this.props.type) {
+      case 'Extended color light':
+        return this.renderLightService();
+      case 'ipcamera':
+        return this.renderGenericService('Camera', 'ip-camera');
+      default:
+        return this.renderGenericService();
+    }
```

Comment:
We probably want to refactor that on the model of `views/services/` and have `views/services/ligh-list-item.jsx` and `views/services/camera-list-item.jsx`.
But this can be done later.